tests/unit/bokeh/plotting/test_figure.py

```diff
@@ -251,6 +252,19 @@ def test_glyph_method_errors_on_sequence_literals_with_source(self) -> None:
         assert m is not None
         assert set(m.groups()) == {"fill_color", "hatch_color", "line_color"}
 
+    def test_context_menu(self) -> None:
```

Comment:
This test passes if copied to the dev branch and run. If you want this test to fail without your code changes, we need to add at least one additional assertion:

```suggestion
    def test_context_menu(self) -> None:
        plot = bpf.figure(context_menu=None)
        assert plot.context_menu == None
        
```

Reply:
Indeed that's the most important one.